hub/core/chunk/chunk_compressed_chunk.py

```diff
@@ -1,29 +1,148 @@
 import numpy as np
-from typing import List
+from typing import List, Union, Sequence
 from hub.core.compression import (
     compress_bytes,
     compress_multiple,
     decompress_bytes,
     decompress_multiple,
 )
+from hub.core.sample import Sample  # type: ignore
+from hub.core.fast_forwarding import ffw_chunk
 from hub.core.serialize import bytes_to_text, check_sample_shape
 from hub.core.tiling.sample_tiles import SampleTiles
 from hub.util.casting import intelligent_cast
 from hub.util.compression import get_compression_ratio
 from hub.util.exceptions import SampleDecompressionError
 from .base_chunk import BaseChunk, InputSample
+from hub.core.serialize import infer_chunk_num_bytes
 
 
 class ChunkCompressedChunk(BaseChunk):
+    def __init__(self, *args, **kwargs):
+        super(ChunkCompressedChunk, self).__init__(*args, **kwargs)
+        if self.is_byte_compression:
+            self.decompressed_bytes = bytearray(
+                decompress_bytes(self._data_bytes, self.compression)
+            )
+        else:
+            shapes = [
+                self.shapes_encoder[i] for i in range(self.shapes_encoder.num_samples)
+            ]
+            self.decompressed_samples = decompress_multiple(self._data_bytes, shapes)
+        self._changed = False
+        self._compression_ratio = 2
```

Comment:
use the get_compression_ratio method instead (it will return 0.5 by default instead of 2 btw)

Reply:
This ratio doesn't have to depend on the compression, can always start with 2.